src/environments/environment.test.ts

```
@@ -29,6 +29,8 @@ const ADMIN_API = `${adminIP}adminapi-v1.0/`;
 const ECD_API = `${ecdIP}ecdapi-v1.0/`;
 const biologicalScreeningDeviceAPI = `${ADMIN_API}diagnostics/biologicalScreeningDevice`; 
 const sessionStorageEncKey = '';
+const fhirIP = 'http://amritwprdev.piramalswasthya.org:8080/';
+const FHIR_API = `${fhirIP}fhirapi-v1.0/`;
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)